visualization/app/codeCharta/codeCharta.model.ts

```diff
@@ -80,6 +87,7 @@ export interface DynamicSettings {
 	searchPattern: string
 	margin: number
 	colorRange: ColorRange
+	searchPanelMode: SearchPanelMode
```

Comment:
This should actually be inside appSettings, as it is not related to the map. But as we wanna split the settings object step by step, should we add a separate service called `searchPanelModeService`, which would have a similar structure to the `loadingGifService`?